packages/ui/src/views/unified-pipeline-studio/components/palette-drawer/components/step-palette-item-layout.tsx

```diff
@@ -1,3 +1,5 @@
+import { cx } from 'class-variance-authority'
```

Comment:
Why not use `import { cn } from '@harnessio/ui/utils'`?

Reply:
done